Assets/Scripts/Interaction Scripts/ControllerInput.cs

```diff
@@ -0,0 +1,348 @@
+﻿namespace ISAACS_ControllerInput {
```

Comment:
Are we still using namespace ISAACS? If so, can we have this nested like:

```suggestion
﻿namespace ISAACS{
﻿    namespace ControllerInput {
```

Reply:
@apollovision

Reply:
But yes we're using namespace ISAACS (which we should clean up at some point) for now

Reply:
Controller Interactions has been phased out, it'll be deleted :)